tf2onnx/tfonnx.py

```diff
@@ -1197,21 +1197,43 @@ def fused_batchnorm_op7(ctx, node, name, args):
 
 
 def matmul_op(ctx, node, name, args):
+    # onnx matmul only support float16, float, double for now
```

Comment:
I understand the newly added logic just can handle non-complex data, so maybe firstly we can check whether "adjoint_a", "adjoint_b"  appears, if yes, then we check the data type (is complex, then throw, otherwise, do the new logic); if no, then your new logic will not run.